commit_build.bat

```diff
@@ -3,4 +3,4 @@ rem code quality checks: yes (findbugs, pmd, checkstyle)
 rem deploy: no deploy
 rem intent of using: production
 rem minification: yes
-mvn clean verify -Pprod
+mvn clean verify -Pprod %1
```

Comment:
Using thos parameter we can pass argument "-Dprotractor.port=8283" for eighty_develop_builder jenkins job